src/ducks/Studio/timeseries/fetcher.js

```diff
@@ -85,8 +87,23 @@ const transformAliases = [
   'percentOfTokenSupplyOnExchanges'
 ]
 
+SOCIAL_TWITTER_INTERVALS.forEach(interval => {
+  const key = 'twitter_followers_' + interval
+
+  Fetcher[key] = {
+    query: GET_METRIC_CHANGES('twitter_followers'),
+    preTransform: aliasTransform('twitter_followers', key),
+    strictVariables: {
+      interval
+    }
+  }
+
+  transformAliases.push('twitter_followers_' + interval)
+})
```

Comment:
This will not be needed after using correct graphql query (comment `[D-0-1]`)

Reply:
Agree